databricks/koalas/indexes.py

```diff
@@ -961,6 +961,91 @@ def max(self):
 
         return result if len(result) > 1 else result[0]
 
+    def argmax(self):
+        """
+        Return a maximum argument indexer.
+
+        Parameters
+        ----------
+        skipna : bool, default True
+
+        Returns
+        -------
+        maximum argument indexer
+
+        Examples
+        --------
+        >>> kidx = ks.Index([10, 9, 8, 7, 100, 5, 4, 3, 100, 3])
+        >>> kidx
+        Int64Index([10, 9, 8, 7, 100, 5, 4, 3, 100, 3], dtype='int64')
+
+        >>> kidx.argmax()
+        4
+        """
+        max_idx = self.max()
+        sdf = self._internal.sdf
+        sdf = sdf.select(self._scol.alias('__index_value__'))
+
+        sdf = _InternalFrame.attach_default_index(
+            sdf, default_index_type='distributed-sequence')
+        # sdf here looks like below
+        # +-----------------+---------------+
+        # |__index_level_0__|__index_value__|
+        # +-----------------+---------------+
+        # |                0|             10|
+        # |                4|            100|
+        # |                2|              8|
+        # |                3|              7|
+        # |                6|              4|
+        # |                5|              5|
+        # |                7|              3|
+        # |                8|            100|
+        # |                1|              9|
+        # +-----------------+---------------+
+
+        sdf = sdf.where(scol_for(sdf, '__index_value__') == max_idx)
```

Comment:
@itholic, can we do this in a single job like the below?

```python
sdf.orderBy(
    F.col("__index_value__").asc(), F.col("__natural_order__").desc()
).select("__index_level_0__").first()[0]
```

Reply:
BTW, it's just an example. Use, for instance, `NATURAL_ORDER_COLUMN_NAME`.

Reply:
i seems possible. thanks for the comment i just fixed!